hooks/log_metrics.py

```diff
@@ -8,43 +8,17 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
-"""Hook that gets executed every time Toolkit logs user metrics."""
+"""
+This hook is deprecated and shouldn't be used anymore.
```

Comment:
why are we deprecating this? I know clients use this hook - for example EA, IIRC to hook into their own metrics stack. Some background here would be great!

Reply:
There will be a new method in this file that accepts the new style metrics. We thought that was preferable over all of a sudden sending the new style metrics to this method. I'm guessing that method will be added shortly.

Reply:
ah! i see! makes perfect sense!!!! 

In that case we should probably tweak the message above a bit^

Reply:
Can we add the new method right away and reference it in the the comment? What name work for you guys? Does it require change somewhere else in toolkit?